lib/Console/Traits/Parallelization.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Console\Traits;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Command\LockableTrait;
+use Symfony\Component\Console\Input\Input;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Webmozarts\Console\Parallelization\Parallelization as WebmozartParallelization;
+
+trait Parallelization
+{
+    use LockableTrait;
+    
+    use WebmozartParallelization
+    {
+        WebmozartParallelization::configureParallelization as parentConfigureParallelization;
+    }
+
+    protected static function configureParallelization(Command $command): void
+    {
+        $command
+            ->addArgument(
+                'item',
+                InputArgument::OPTIONAL,
+                'The item to process'
```

Comment:
This is not only for internal use. The argument can be used to pass specific IDs for items that will be generated

Reply:
So it is a "hidden feature". For the image thumbnail command it could be used to render thumbnails for specific images. For other commands where the (serialized) items are more complex it is probably not of use.

Reply:
I can add that to the description.